packages/cli-types/src/node-stream-zip.d.ts

```diff
@@ -0,0 +1,66 @@
+// https://github.com/antelle/node-stream-zip/issues/36#issuecomment-596249657
```

Comment:
Maybe it would be worth to contribute those typings to `@DefinitelyTyped`? :P

Reply:
I had to do some modifications from the code in that comment and the discussion in that thread made me believe they were going to be added eventually in the package.